### PR TITLE
fix(965):[2] $GIT_BRANCH invalid value in PR builds 

### PR DESCRIPTION
**Pull request by ibu1224**
## Context

I fix issue.
I add new variables,`PR_BRANCH_NAME`

This PR is requires below changes before merge.
https://github.com/screwdriver-cd/data-schema/pull/386

## Objective
In this PR, get `prBranchName`

## License

I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

**Comment by ibu1224** [comment]
The reason test is failing is because it has not been merged.
https://github.com/screwdriver-cd/data-schema/pull/386

File: test/index.test.js [hunk 1]
```diff
@@ -656,7 +680,8 @@ describe('index test', () => {
             owner: 'owner',
             repo: 'repo',
             ref: 'master',
-            scmContext: 'github:github.com'
+            scmContext: 'github:github.com',
+            refType: 'heads'
```
Comment (kumada626) [inline test/index.test.js]: Why add `refType` here? I think that the change in index.js does not affect this test case.

Comment (ibu1224) [reply test/index.test.js]: Because the branch was not up to date.
so I took in origin/master

Comment (ibu1224) [reply test/index.test.js]: Oh. sorry
I forgot to fix.

**Pull request merged**